docs/dev/policy_documents/standards-for-tests.md

```diff
@@ -176,33 +178,25 @@ fi
 
 TEST_PKG_IDENT="${1}"
 export TEST_PKG_IDENT
+
 hab pkg install core/bats --binlink
 hab pkg install "${TEST_PKG_IDENT}"
-bats "$(dirname "${0}")/test.bats"
+bats "${TESTDIR}/test.bats"
 ```
 
 BATS test script `7zip/tests/test.bats`:
 
 ```bash
-TEST_PKG_VERSION="$(echo "${TEST_PKG_IDENT}" | cut -d/ -f3)"
-@test "7z exe runs" {
-  run hab pkg exec ${TEST_PKG_IDENT} 7z
-  [ $status -eq 0 ]
-}
-
-@test "7z version matches ${TEST_PKG_VERSION}" {
-  run hab pkg exec ${TEST_PKG_IDENT} 7z
-  [[ "$output" =~ 7-Zip\ \[64\]\ ${TEST_PKG_VERSION} ]]
-}
+expected_version="$(echo "${TEST_PKG_IDENT}" | cut -d/ -f3)"
 
-@test "7za exe runs" {
-  run hab pkg exec ${TEST_PKG_IDENT} 7za
-  [ $status -eq 0 ]
+@test "7z version matches ${expected_version}" {
+   run hab pkg exec "${TEST_PKG_IDENT}" 7z
```

Comment:
Is 3 spaces instead of 2 here intentional, and is shellcheck OK with it?

Reply:
Good eyes catching those spaces Stocksy!  Fixed.